### PR TITLE
Add peru prescription configuration

### PR DESCRIPTION
**Pull request by MiguelAHPpih**
Depends on https://github.com/PIH/openmrs-module-pihcore/pull/325 and https://github.com/PIH/openmrs-config-ses/pull/48

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -900,7 +913,7 @@ private void enableDispensing() {
                 sessionLocationHasTag("Dispensing Location")));
 
         extensions.add(visitAction(Extensions.DISPENSE_MEDICATION_VISIT_ACTION,
-                "dispensing.app.label",
+                "Receta",
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: Just create a message override for `dispensing.app.label`.

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 2]
```diff
@@ -449,6 +449,9 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
         if (config.isComponentEnabled(Components.REHAB)) {
             enableRehab();
         }
+        if(config.isComponentEnabled(Components.PERU_PRESCRIPTIONS)){
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:452]:
```suggestion
        if (config.isComponentEnabled(Components.PERU_PRESCRIPTIONS)) {
```

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java (hunk 1 above)
Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: Agree, yes, we don't want to hard-code a name here....

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java (hunk 2 above)
Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:452]: Let's rename this component to "Prescription" and just put a comment "currently only used in Peru" here.

**Review by mogoodrich — COMMENTED** [review]
LGTM, with the general suggestion that I think we can just name this "Prescription"

**Review by brandones — APPROVED** [review]

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java (hunk 1 above)
Comment (brandones) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: Ping @MiguelAHPpih , this is actually important

Comment (MiguelAHPpih) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: @brandones I'm not using that method.

Comment (brandones) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: Using what method? This dispensing app is used by several sites; we can't have it show up as "Receta" in Haiti.

Comment (MiguelAHPpih) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:916]: Oh i think this was from when i tried to use the dispensing component, I'll leave it as it was.

Comment (brandones) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: Perfect, thanks

**Review by brandones — APPROVED** [review]

**Pull request merged**